resources/styles.css

```diff
@@ -71,80 +85,80 @@ table.dataTable tbody tr td.proj_name {
   font-weight: bold;
 }
 
-table.dataTable.display tbody tr.active.odd {
```

Comment:
If I understand correctly, this removes the special `active` class.
Granted, it's not very clear :-) , but this class is actually [used](https://github.com/c4dt/showcase/blob/main/views/projects.tpl#L190) to mark the "active" projects in green (i.e. the ones whose last commit is within the last 9 months). On https://showcase.c4dt.org/, you can see that the projects towards the bottom should be mostly grey. That is unfortunately what makes the table CSS classes a bit cumbersome, as there are the alternate greys (even/odd) and the alternate greens (even/odd), in addition to the sorting versions...